src/test/java/com/answerdigital/answerking/service/OrderServiceTest.java

```diff
@@ -222,20 +239,21 @@ void testUpdateOrderWithInvalidOrderIdThrowsNotFoundException() {
     void testUpdateOrderWithAlreadyCancelledOrderThrowsOrderCancelledException() {
         // Given
         final Order order = orderTestBuilder
-                .withDefaultValues()
-                .withOrderStatus(OrderStatus.CANCELLED)
-                .build();
+            .withDefaultValues()
+            .withId(1L)
```

Comment:
should orderTestBuilder's 'withDefaultValues' not make an Order with ID=1L by default? Trying to avoid magic values as much as possible.